int8 mass_status

int8 EMPTY=0
int8 BOTTLE=1